futures-util/src/try_stream/try_for_each.rs

```diff
@@ -6,7 +6,7 @@ use pin_utils::{unsafe_pinned, unsafe_unpinned};
 
 /// Future for the [`try_for_each`](super::TryStreamExt::try_for_each) method.
 #[derive(Debug)]
-#[must_use = "streams do nothing unless polled"]
```

Comment:
I'd still expect the streams to say "streams do nothing ... "

Reply:
`TryForEach` implements the future, not the stream: https://rust-lang-nursery.github.io/futures-api-docs/0.3.0-alpha.16/futures/stream/struct.TryForEach.html#impl-Future

(The types that implement the stream are keeping that description.)

Reply:
hoh boy thank you